style.css

```diff
@@ -340,6 +340,90 @@ body {
 
 /* END OF ABOUT SECTION STYLING */
 
+/* Beginnings of contact section */
+.form-section {
+  background-color: #6070ff;
+  background-image: url(./images/contact-form.png);
+  height: 600px;
+  display: flex;
+  align-items: center;
+  border-top-left-radius: 5rem;
+  box-shadow: 2px 4px 4px rgba(0, 0, 0, 0.1);
+  flex-direction: column;
+  padding: 7rem 2rem 2rem 2rem;
+  font-family: poppins, sans-serif;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
```

Comment:
- [ ] Ouch this is close but you seem to be missing 1 icon from the background image of the contact section as shown below in the original design 👇 please make sure to pay attention to small details like this as this is the trait of a great frontend developer.
- Figma design:
  ![image](https://github.com/blessing-michael/portfolio/assets/104968236/39ad1394-607b-46be-bc89-a5324fc40fef)
- Your design:
  ![image](https://github.com/blessing-michael/portfolio/assets/104968236/9a7715c2-28a5-4b85-b661-0083bfa8098d)